local-volume/README.md

```diff
@@ -98,30 +98,81 @@ $ ALLOW_PRIVILEGED=true LOG_LEVEL=5 FEATURE_GATES=PersistentLocalVolumes=true ha
+2. Provisioner's default values.yaml which contains variables used for rendering a template.
+3. (Optional) User's customized values.yaml as a part of helm template command. User's provided
+   values will override default values of Provisioner's values.yaml.
+
+Below is the default values.yaml, user should not remove variables from this file but can change any values of these variables. 
+The only exception is the list of storage classes and respective mounting points. In general there should be at least one storage class
+listed with its corresponding "hostDir", "mountDir" parameter is optional in case and in case of absense, the value of "defaultMountDir" will be used.
+
+```
+common:
+  namespace: default
+  defaultHostDir: /mnt/disks
+  defaultMountDir: /local-disks
+configmap:
+  configMapName: "test"
+#
+# Storage Class part should be customized according to a user's
+# actual local volumes configuration.
+#
+  storageClass:
+    - local-fast:
+        hostDir: "/mnt/fast"
+        mountDir: "/local-fast"
+    - local-normal:
+        hostDir: "/mnt/normal"
+        mountDir: "/local-normal"
+    - local-slow:
+        hostDir: "/mnt/slow"
+        mountDir: 
+daemonset:
+  name: "local-volume-provisioner"
+  image: quay.io/external_storage/local-volume-provisioner:latest
+  imagePullPolicy: Always
+```
+In the simplest form, to generate Provisioner spec, you run:
+
+``` console
+helm template local-volume/helm/provisioner > provisioner.yaml
+```
+
+This command will generate a ready to use with **kubectl** yaml file. A user can further
+modify generated file, but then **kubectl** needs to be used to check validity of additional changes.
+
+In case a user  does not want to modify default values.yaml, a custom yaml file can be used in the command line:
 
-1. Create an admin account with cluster admin priviledge:
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/admin-account.yaml
+helm template local-volume/helm/provisioner --values custom-values.yaml > provisioner.yaml
 ```
 
-2. Create a ConfigMap with your local storage configuration details.
+In order to be able to use **helm** to render templates, it has to be installed on a host where a user plans
+to generate templates.
+
+These two commands will download and copy helm binary into /usr/bin directory.
+
 ``` console
-$ kubectl create -f bootstrapper/deployment/kubernetes/<version>/example-config.yaml
+export HELM_URL=http://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz
+curl "$HELM_URL" | sudo tar --strip-components 1 -C /usr/bin linux-amd64/helm -zxf -
 ```
-**Note**: This configmap is required in order to run the bootstrapper.
 
+Provisioner's spec generation process has been tested with helm version 2.7.2.
 
-3. Launch the bootstrapper, which in turn creates static provisioner daemonset:
+Once a user is satisfied with the content of Provisioner's yaml file, **kubectl** can be used
```

Comment:
Once the specs are generated, create the configmap and provisioner:

Reply:
done